Material.Styles/Themes/MaterialThemeBase.cs

```diff
@@ -39,34 +39,71 @@ public MaterialThemeBase(IServiceProvider serviceProvider)
 
         private IResourceDictionary LoadedResourceDictionary => (Loaded as Avalonia.Styling.Styles)!.Resources;
 
-        public static readonly DirectProperty<MaterialThemeBase, ITheme> CurrentThemeProperty =
-            AvaloniaProperty.RegisterDirect<MaterialThemeBase, ITheme>(
+        public static readonly DirectProperty<MaterialThemeBase, ITheme?> CurrentThemeProperty =
```

Comment:
It is assumed that CurrentTheme always has a value

Reply:
I pushed some commits, including remove nullable annotations